package.json

```diff
@@ -37,8 +37,8 @@
     "minimatch": "^3.0.0",
     "progress": "^1.1.8",
     "shelljs": "^0.5.3",
-    "typedoc-default-themes": "0.4.0",
-    "typescript": "1.7.3"
+    "typedoc-default-themes": "sierrasoftworks/typedoc-default-themes#v0.3.5",
```

Comment:
@blakeembrey Keep in mind that this is still in here; make sure that you change this to reference the official module before you do a release.